aten/src/ATen/core/boxing/KernelFunction.cpp

```diff
@@ -49,4 +49,25 @@ bool KernelFunction::_equalsBoxedAndUnboxed(const KernelFunction& other) const {
          unboxed_kernel_func_ == other.unboxed_kernel_func_;
 }
 
+void KernelFunction::checkBoxedKernel(const OperatorHandle& opHandle) const {
+  if (C10_UNLIKELY(boxed_kernel_func_ == nullptr)) {
+      if (unboxed_kernel_func_ == nullptr) {
+          TORCH_INTERNAL_ASSERT(
+              false,
+              "Tried to call KernelFunction::callBoxed() on an uninitialized KernelFunction.",
+              " opname: ",
+              opHandle.operator_name(),
+              " For mobile selective build please make sure to include all ops exported from `torch.jit.export_opnames(model)`.");
```

Comment:
If somebody who doesn't know what a selective build is sees this, it might be confusing. Maybe say "If you're using mobile selective build" instead of "For mobile selective build" to make it more clear.